proposals/community-strategy/README.md

```diff
@@ -0,0 +1,34 @@
+# Community engagement strategy
+
+The community is the key for a standard. Without adoption, the defined standard is pointless.
+Other standards, such as [Project Connected Home over IP](https://www.connectedhomeip.com/), 
+are moving forward with a lot of support from the industry. 
+
+This directory contains ideas how to push the standard and grow the community behind it.
+
+## 1. Documentation and libraries
+
+Before reaching out to a wide range of developers the base must be present.
+Reference implementations in the major languages, such as C, Javascript and Java, must be present and well maintained.
+An extensive documentation for them must be present as well.
+
+## 2. Reaching out to Open source projects
+
+Integration into other projects is very important. 
+Therefore it makes sense to get in touch with relevant Open source communities.
+In order to help them with their implementations, people must be present on relevant [social media channels](social-media.md).
```

Comment:
True, there is. I also have a [couple of follower on Twitter](https://twitter.com/Citrullin). Tbh: I am not sure if Twitter is the right tool for a standard and the W3C already has a twitter account. So, I am not sure if it would be worth to create an account just for a standard. I mean, it is worth a try, right? ^^ Someone just needs to maintain it. It doesn't make sense to have an inactive account.

Reply:
Actually, we have [it](https://twitter.com/W3C_WoT) :) I think it is managed by @egekorkan or @sebastiankb.  It is mainly oriented to dissemination than discussion. I think it is pretty useful.